lib/elixir/lib/kernel.ex

```diff
@@ -3271,7 +3271,12 @@ defmodule Kernel do
       if Map.has_key?(fields, :message) do
         @spec message(Exception.t) :: String.t
         def message(exception) do
-          exception.message
+          cond do
+            is_nil(exception.message) ->
```

Comment:
Anyway, blank error message won't help understand the reason or error, thus we shouldn't help people have blank error messages. :bowtie: